seacrowd/sea_datasets/limesoda/limesoda.py

```diff
@@ -0,0 +1,171 @@
+import json
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@INPROCEEDINGS{9678187,
+  author={Payoungkhamdee, Patomporn and Porkaew, Peerachet and Sinthunyathum, Atthasith and Songphum, Phattharaphon and Kawidam, Witsarut and Loha-Udom, Wichayut and Boonkwan, Prachya and Sutantayawalee, Vipas},
+  booktitle={2021 16th International Joint Symposium on Artificial Intelligence and Natural Language Processing (iSAI-NLP)},
+  title={LimeSoda: Dataset for Fake News Detection in Healthcare Domain},
+  year={2021},
+  volume={},
+  number={},
+  pages={1-6},
+  doi={10.1109/iSAI-NLP54397.2021.9678187}}
+"""
+
+_DATASETNAME = "limesoda"
+
+_DESCRIPTION = """\
+Thai fake news dataset in the healthcare domain consisting of curate and manually annotated 7,191 documents
+(only 4,141 documents contain token labels and are used as a test set of the baseline models).
+Each document in the dataset is classified as fact, fake, or undefined.
+"""
+
+_HOMEPAGE = "https://github.com/byinth/LimeSoda"
+
+_LICENSE = Licenses.CC_BY_4_0.value
+
+_LANGUAGES = ["tha"]
+
```

Comment:
can you add constant `_LOCAL = False`? it indicates this dataloader doesn't need an indirect download (which can't be catered by HF `DownloadManager` methods).
```suggestion
_LANGUAGES = ["tha"]
_LOCAL = False

```